Formula/snowpack.rb

```diff
@@ -19,7 +19,19 @@ class Snowpack < Formula
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bin.install_symlink Dir[libexec/"bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = "darwin"
+    on_linux { os = "linux" }
```

Comment:
```suggestion
    os = OS.mac? ? "darwin" : "linux"
```
I think we've decided it's okay to use `OS.mac?` in `install` methods now, so let's make this a bit less ugly.

Reply:
👍